velox/common/encode/Base64.h

```diff
@@ -110,6 +110,16 @@ class Base64 {
       char* outputBuffer,
       size_t outputSize);
 
+  /// Encodes the input buffer into Base64 MIME format.
+  /// Inserts a CRLF every kMaxLineLength output characters.
+  static void
+  encodeMime(const char* input, size_t inputSize, char* outputBuffer);
+
+  /// Decodes a Base64 MIME‐mode buffer back to binary.
+  /// Skips any non-Base64 chars (e.g. CR/LF).
+  static Status
+  decodeMime(const char* input, size_t inputSize, char* outputBuffer);
```

Comment:
Do we need to add them in the common lib if only used by sparksql functions?

Reply:
These functions’ implementations depend on variables defined in the Base64.cpp file, for example: `kBase64Charset`, and RFC2045 is part of the Base64 standard, so puting them together might be more reasonable.